### PR TITLE
Hosting dashboard: clean up monitoring and logs tabs

### PR DESCRIPTION
**Pull request by fushar**
Part of:

- https://github.com/Automattic/dotcom-forge/issues/7924

## Proposed Changes

This PR refactors the code for the hosting dashboard's Monitoring and Logs tabs. Previously, they were under the `client/my-sites/site-monitoring/` directory, because they were once in a single page in site-specific view (not global view).

Now, this PR cleans those up by splitting them into separate `client/hosting/monitoring/` and client/hosting/logs/` directories.

## Why are these changes being made?

Cleaning up code to avoid future confusion.

## Testing Instructions

1. Go to `/sites`.
2. Click an Atomic site.
3. Smoke test the Monitoring and Logs tabs. Make sure the features still work.

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [x] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

**Comment by fushar** [comment]
I think every open PR has this test failing.

File: client/hosting/logs/index.tsx [hunk 1]
```diff
@@ -10,8 +10,8 @@ import {
 } from 'calypso/hosting/sites/components/site-preview-pane/constants';
 import { siteDashboard } from 'calypso/hosting/sites/controller';
 import { siteSelection, sites, navigation } from 'calypso/my-sites/controller';
-import { redirectHomeIfIneligible } from 'calypso/my-sites/site-monitoring/controller';
-import { httpRequestLogs, phpErrorLogs } from './controller';
+import { redirectHomeIfIneligible } from '../monitoring/controller';
+import { phpErrorLogs, webServerLogs } from './controller';
 
 export default function () {
```
Comment (nightnei) [inline client/hosting/logs/index.tsx:16]: Just out of curiosity - who run this function, to initiate routes?

**Review by nightnei — APPROVED** [review]
I haven't tested, but the code LGTM, thanks for doing it  👍

UPDATE: tested, works as expected

File: client/hosting/logs/index.tsx (hunk 1 above)
Comment (fushar) [reply client/hosting/logs/index.tsx:16]: Should be here:

https://github.com/Automattic/wp-calypso/pull/94524/files#diff-bbfd0752d7fc9a4e930a58d3f153a24662dd06baa00a529caf8b7715e9477003R737

**Pull request merged**